downstream/modules/troubleshooting-aap/proc-troubleshoot-use-in-controller.adoc

```diff
@@ -0,0 +1,53 @@
+[id="troubleshoot-use-in-controller"]
+= Issue - Cannot select the "Use in Controller" option for {ExecEnvShort} image on {PrivateHubName}
+
+You cannot use the *Use in Controller* option for an {ExecEnvShort} image on {PrivateHubName}. You also receive the following error message: “No Controllers available”.
```

Comment:
Nitpicky, but I would say "you also receive the ~~following~~ error message "No Controllers available".